### DISCUSSION TITLE
libSQL DB

### DISCUSSION THREAD

**Discussion by akawahuynh** [coollabsio/coolify#3814] [Database Requests]
Add libSQL db a one click database.

**Comment by candidia** [comment]
Here're the official docker instructions https://github.com/tursodatabase/libsql/blob/main/docs/DOCKER.md for reference

**Comment by Deveopix** [comment]
We need thissss.

**Comment by Liangyiu** [comment]
Just for anyone stumbling over this while searching for a way to selfhost libsql.
Found this article [Self-hosting Turso libSQL](https://hubertlin.me/posts/2024/11/self-hosting-turso-libsql) on the topic, it has a section about coolify and how to set it up :smile:

**Comment by rober-tm** [comment]
Seria excelente añadir el templete a la parte de bases de datos, hay posibilidad de incorporarlo???
